benefits/enrollment/views.py

```diff
@@ -80,7 +80,11 @@ def index(request):
 
         if verifier.eligibility_confirmed_item_heading or verifier.eligibility_confirmed_item_details:
             heading = _(verifier.eligibility_confirmed_item_heading) if verifier.eligibility_confirmed_item_heading else None
-            details = _(verifier.eligibility_confirmed_item_details) if verifier.eligibility_confirmed_item_details else None
+            details = (
+                _(verifier.eligibility_confirmed_item_details) % {"transit_agency_short_name": agency.short_name}
```

Comment:
Does this need a call to `format_html`?

Reply:
There's no HTML in the value, so no need to call `format_html`
```
"Your eligibility has been verified by Login.gov. This benefit will give you "
"a half-price fare on public transportation with "
"%(transit_agency_short_name)s."
```